src/app/shared/components/general-table/general-table.component.html

```diff
@@ -17,6 +17,9 @@
                             </th>
                         </tr>
                         <tr *ngIf="config.showColumnTitle">
+                            <ng-container *ngIf="config.sequence">
+                                <th scope="col" [class]="config.sequence.title.class">{{config.sequence.title.name}}</th>
+                            </ng-container>
```

Comment:
En teoría basta con una sola etiqueta...
```suggestion
                            <th scope="col" [class]="config.sequence.title.class" *ngIf="config.sequence">
                                {{config.sequence.title.name}}
                            </th>
```
(En general, aplicar [KISS](https://www.blogotec.com/principios-en-el-diseno-del-software-kiss-dry-y-solid/), optar por la menor cantidad posible de código...)